package/android/src/main/java/com/mrousavy/camera/types/Events.kt

```diff
@@ -0,0 +1,36 @@
+package com.mrousavy.camera.types
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.uimanager.events.Event
+
+class CameraInitializedEvent(surfaceId: Int, viewId: Int) : Event<CameraInitializedEvent>(surfaceId, viewId) {
+  override fun getEventName() = "cameraInitialized"
+  override fun getEventData(): WritableMap = Arguments.createMap()
```

Comment:
Does this have to be an empty map, or can this also be `null`? What's the recommended approach here?

Reply:
It was required not to be a null from 0.65 to 0.73. From 0.74 it can be null again.